app/views/layouts/application.html.erb

```diff
@@ -11,5 +11,6 @@
 
   <body>
     <%= yield %>
+    <script src="https://d3js.org/d3.v5.min.js"></script>
```

Comment:
You should be able to avoid doing this by importing d3 into the relevant React files with

`import * as d3 from "d3";`